README.md

```diff
@@ -1,20 +1,51 @@
-# Turing.jl
+<p align="center"><img src="https://raw.githubusercontent.com/TuringLang/turinglang.github.io/refs/heads/main/assets/images/turing-logo.svg" alt="Turing.jl logo" width="200" /></p>
+<h1 align="center">Turing.jl</h1>
 
-[![Build Status](https://github.com/TuringLang/Turing.jl/workflows/Turing-CI/badge.svg)](https://github.com/TuringLang/Turing.jl/actions?query=workflow%3ATuring-CI+branch%3Amain)
-[![Coverage Status](https://coveralls.io/repos/github/TuringLang/Turing.jl/badge.svg?branch=main)](https://coveralls.io/github/TuringLang/Turing.jl?branch=main)
-[![codecov](https://codecov.io/gh/TuringLang/Turing.jl/branch/main/graph/badge.svg?token=OiUBsnDQqf)](https://codecov.io/gh/TuringLang/Turing.jl)
-[![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor%27s%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
+**Turing.jl** is a framework for probabilistic programming and Bayesian inference in Julia.
 
-## Getting Started
+## 📚 Documentation
 
-Turing's home page, with links to everything you'll need to use Turing, is available at:
+**https://turinglang.org** contains the documentation for the broader TuringLang ecosystem.
 
-https://turinglang.org/docs/
+**https://turinglang.org/Turing.jl/** specifically contains the API documentation for anything exported by Turing.jl.
 
-## What's changed recently?
+## 🛠️ Contributing
 
-See [releases](https://github.com/TuringLang/Turing.jl/releases).
+### Issues
 
```

Comment:
What something like this still be useful, linking to releases and/or HISTORY.md?